### DISCUSSION TITLE
HA set up issues

### DISCUSSION THREAD

**Discussion by thechaps** [mrlt8/docker-wyze-bridge#1345] [Q&A]
I've followed the setup instructions on this site and multiple videos on YT... I can't seem to get the RTSP side to work. After tweaking settings in the config I finally got the 2 cams I have to show up in the app in HA. The main problem I have now is the streams are setting the URL as homeassistant.local instead of HA's IP like the instructions/videos i've seen say it should.

![image](https://github.com/user-attachments/assets/ea8c7118-d9e5-45ec-b5d9-f21afe36f63a)

I've set the bridge to the IP of HA but these URLs stay with homeassistant.local.

My HA is running on a VM inside Unraid.

The main reason I need this to be an IP is I have agent DVR set up on Unraid to record these cameras too.

Any help would be great!

**Comment by mrlt8** [comment]
You should be able to just replace the `homeassistant.local` portion of the address to match your server's IP.

Note that you may also need to configure or disable authentication for the stream.
